src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -42,6 +42,8 @@
 #[AsCommand(name: 'translation:extract', description: 'Extract missing translations keys from code to translation files.')]
 class TranslationUpdateCommand extends Command
 {
+    use TranslationTrait;
```

Comment:
depending on an internal trait of another bundle looks like a bad idea. It would force us to stop using semver constraints in the composer.json, as the trait is not covered by semver (and so changing it in Symfony 6.2 might break FrameworkBundle 6.1)